mir/src/node.rs

```diff
@@ -227,10 +227,9 @@ impl MirNode {
             // otherwise, just look up in the column set
             _ => match self.columns
                 .iter()
-                .position(|cc| cc.name == c.name && cc.table == c.table)
+                .position(|cc| cc.name == c.name)
```

Comment:
Is this relaxation potentially dangerous? Seems like we could end up with the wrong column if two tables we join had the a column of the same name. (note to self: check with `git blame` why we added this.)